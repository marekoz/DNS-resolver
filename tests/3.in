dns.google.com -r -s kazi.fit.vutbr.cz