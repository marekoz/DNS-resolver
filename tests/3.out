Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
  dns.google.com., Type: A, Class: IN
Answer section (2)
  dns.google.com., Type: A, Class: IN, TTL: <number>, Data length: 4, 8.8.4.4
  dns.google.com., Type: A, Class: IN, TTL: <number>, Data length: 4, 8.8.8.8
Authority section (0)
Additional section (0)